src/correctness.ts

```diff
@@ -0,0 +1,110 @@
+// import { Octokit } from "octokit";
+import { Octokit as OctokitType } from "octokit";
+const Octokit = OctokitType as any;
+import * as readline from 'readline';
+
+// Set up authentication token
+// To do this, I copied and pasted my github token into an environment variable
+// in vs code and referenced it in this line
+const octokit = new Octokit({ 
+  auth: "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",
```

Comment:
You should for sure take this token out of the file itself because I think it could get stolen if it's just in plain text like this. Add it as an environment variable.

Reply:
Fixed.